cmd/killTask.go

```diff
@@ -0,0 +1,74 @@
+/**
```

Comment:
We dont need this file, please add a sub command for kill task in kill command. 
So, most of the code must be moved to kill.go